### PR TITLE
Update architect orb/makefile/gh workflows latest versions

### PR DESCRIPTION
**Pull request by axbarsan**
## Checklist

- [ ] Update changelog in CHANGELOG.md. (Not important to the customer)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,39 +64,58 @@ $(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-%-amd64.tar.gz: $(APPLICATION)-v$(VERS
...
+	@echo "====> $@"
 	go test -ldflags "$(LDFLAGS)" -race ./...
 
 .PHONY: build-docker
 ## build-docker: builds docker image to registry
 build-docker: build-linux
+	@echo "====> $@"
 	docker build -t ${APPLICATION}:${VERSION} .
 
+.PHONY: lint-chart
```
Comment (ubergesundheit) [inline Makefile:107]: In which context is this bein run? After clone running `make lint-chart` does fail like this on my machine:

```
$ make lint-chart
====> lint-chart
rm -rf /tmp/kubectl-gs-test
mkdir -p /tmp/kubectl-gs-test/helm
cp -a ./helm/kubectl-gs /tmp/kubectl-gs-test/helm/
cp: cannot stat './helm/kubectl-gs': No such file or directory
make: *** [Makefile:114: lint-chart] Error 1
```

Comment (axbarsan) [reply Makefile:107]: Thank you for pointing that out

It's not ran in any way in this case. That's used for apps and operators, I believe, in order to verify that the helm chart is valid.

I opened an issue to only make the script present for the flavours that need it
https://github.com/orgs/giantswarm/projects/163#card-42409142

Comment (kopiczko) [reply Makefile:107]: Does it hurt to have it here?

Comment (ubergesundheit) [reply Makefile:107]: no

Comment (axbarsan) [reply Makefile:107]: It doesn't hurt, but that's why we have different flavours, so we'd be able to only include what we need for each of them

**Pull request merged**